package.json

```diff
@@ -79,5 +78,6 @@
     "extends": [
       "@commitlint/config-conventional"
     ]
-  }
+  },
+  "packageManager": "yarn@1.22.22"
```

Comment:
Why don't you downgrade to 1.22.21 so you don't have this problem anymore

Reply:
It's a feature not a problem and wasn't it agreed we'd standardize it since we are a yarn-only org with the exception of one or two bun repos? If we are no longer standardizing it I'll change my sys config

Reply:
No it's not a problem for anybody except your yarn

Reply:
https://github.com/ubiquity-os-marketplace/automated-merging/pull/19#issuecomment-2345778583
https://github.com/ubiquity-os-marketplace/command-start-stop/pull/1#discussion_r1651178825
https://github.com/ubiquity/cloudflare-deploy-action/issues/2#issue-2170840592
https://github.com/ubiquity/rpc-handler/pull/39#issuecomment-2260304403

I don't understand your comment. It has been a problem multiple times and Mentlegen, Rndquu and myself have openly stated to adopt it, also you and I have had this conversation multiple times and each time previously it appeared like you had agreed so I am confused. Is it a problem or not?

Reply:
I never agreed

Reply:
https://github.com/ubiquity-os-marketplace/disqualifier/pull/15#discussion_r1720523533 You did here which is why I'm so confused. There are about 8 repos across the across the orgs including plugins, ubiquibot etc that have this defined. If I should remove it here should other occurrences be removed as well?